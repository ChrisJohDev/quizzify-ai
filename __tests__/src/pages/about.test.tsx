import {render, screen }from '@testing-library/react';
import About from '@/pages/about';
import '@testing-library/jest-dom'

describe('ID: QAI:base-TC013 - About Page', () => {
  it('renders correctly', () => {
    const { container } = render(<About />);
    expect(container).toMatchSnapshot();
  });
});