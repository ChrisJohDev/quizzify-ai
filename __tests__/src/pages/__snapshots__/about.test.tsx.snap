// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About - snapshot test renders correctly 1`] = `
<div>
  <div
    class="flex flex-col aboutPage"
  >
    <h1
      class="text-center headline"
    >
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
       the Story
    </h1>
    <p
      class="paragraph"
    >
      In the spring of 2023, a team of passionate innovators and trivia enthusiasts saw a gap in the market. They recognized the challenges faced by quiz creators in producing engaging and stimulating questions and the lack of a convenient platform for quiz enthusiasts to participate. The idea was simple - to make quiz creation and participation a breeze. Thus, 
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
       was born.
    </p>
    <p
      class="paragraph"
    >
      After some time of rigorous research, development, and testing, the 
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
       platform was launched. Utilizing state-of-the-art artificial intelligence algorithms, it aimed to ease the pain of QuizMasters by offering a vast database of exciting questions. The response was overwhelming. Quiz Masters, amateur enthusiasts, and even businesses recognized the platform’s potential as a unique tool to engage their audience.
    </p>
    <p
      class="paragraph"
    >
      By the end of 2023, 
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
       will become a prominent name in the world of quiz creation and participation. Its user-friendly interface and the AI-powered engine, coupled with the endless possibilities of quiz customization, allows it to stand out among its competitors. The platform’s reach expands beyond geographical boundaries, connecting trivia enthusiasts worldwide, and fostering a sense of community among its users.
    </p>
    <p
      class="paragraph"
    >
      Fast forward to the present, 
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
       is celebrated as a leading online quizzing platform. It has significantly reshaped the landscape of trivia quizzing and has brought a wave of innovation to this age-old leisure activity.
    </p>
    <h2>
      The Future of 
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
    </h2>
    <p
      class="paragraph"
    >
      As 
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
       looks toward the future, the vision remains as strong as ever: to continuously simplify and enhance the quizzing experience for everyone. The following are the strategic initiatives outlined for the future:
    </p>
    <ol>
      <li>
        <p
          class="paragraph"
        >
          <strong>
            Enhanced Personalization:
          </strong>
           In the future, 
          <span
            class="companyName keepColor"
          >
            Quizzify-AI
          </span>
           plans to incorporate a greater degree of personalization in its platform. This will include adaptive quizzes that adjust difficulty based on player performance and personalized recommendations based on user quiz preferences.
        </p>
      </li>
      <li>
        <p
          class="paragraph"
        >
          <strong>
            Expanded Content Library:
          </strong>
           
          <span
            class="companyName keepColor"
          >
            Quizzify-AI
          </span>
           will continue to grow its vast database of questions, aiming to cover more diverse topics, languages, and difficulty levels.
        </p>
      </li>
      <li>
        <p
          class="paragraph"
        >
          <strong>
            Business Solutions:
          </strong>
           Recognizing the potential of quizzes as a business tool, 
          <span
            class="companyName keepColor"
          >
            Quizzify-AI
          </span>
           intends to offer specialized solutions for businesses to engage their customers or train their employees.
        </p>
      </li>
      <li>
        <p
          class="paragraph"
        >
          <strong>
            Mobile Application:
          </strong>
           To make quizzing even more accessible, 
          <span
            class="companyName keepColor"
          >
            Quizzify-AI
          </span>
           will develop a mobile application that provides the same robust functionality as the web platform.
        </p>
      </li>
      <li>
        <p
          class="paragraph"
        >
          <strong>
            Partnerships:
          </strong>
           
          <span
            class="companyName keepColor"
          >
            Quizzify-AI
          </span>
           will seek partnerships with educational institutions, businesses, and entertainment entities to bring quality quizzes to a broader audience.
        </p>
      </li>
      <li>
        <p
          class="paragraph"
        >
          <strong>
            Community-Building:
          </strong>
           
          <span
            class="companyName keepColor"
          >
            Quizzify-AI
          </span>
           will focus on strengthening its user community, with features like user-created quizzes, community forums, and social media integrations.
        </p>
      </li>
      <li>
        <p
          class="paragraph"
        >
          <strong>
            Advanced Analytics:
          </strong>
           
          <span
            class="companyName keepColor"
          >
            Quizzify-AI
          </span>
           will introduce advanced analytics for Quiz Masters to understand the performance of their quizzes, participant engagement, and much more.
        </p>
      </li>
    </ol>
    <p
      class="paragraph"
    >
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
      ’s journey from a simple idea to a leading online quizzing platform is a testament to its dedication to innovation and its commitment to its users. With these ambitious plans, 
      <span
        class="companyName keepColor"
      >
        Quizzify-AI
      </span>
       looks forward to further revolutionizing the world of quizzing, making it more exciting and accessible for everyone.
    </p>
  </div>
</div>
`;
